tutorials/plot_introduction.py

```diff
@@ -278,7 +278,7 @@
 ##############################################################################
 # Compute a contrast:
 
-contrast = evoked1 - evoked2
+contrast = mne.combine_evoked([evoked1, evoked2], weights=[1, -1])
```

Comment:
can you explicit the weighting parameter and add a note eg reuse the blob of text that you deleted from pitfalls.rst?